artifacts/flagger/crd.yaml

```diff
@@ -852,6 +855,9 @@ spec:
                                 - MUTUAL
                                 - ISTIO_MUTUAL
                               type: string
+                            warmupDurationSecs:
```

Comment:
why is this added to the TLS configuration? i don't see it in the istio docs: https://istio.io/latest/docs/reference/config/networking/destination-rule/#ClientTLSSettings

Reply:
Thank you for pointing this out!
Fixed.
https://github.com/fluxcd/flagger/pull/1540/commits/b70287dabb798f5cbd91cb9e1acdbc259e852470